src/admin/redux/modules/payment_defaults.js

```diff
@@ -39,7 +39,7 @@ const payment_defaults : Reducer<Payment, Action>
   = (state = initial_state, {type, payload}) => {
     switch (type) {
       case FETCHED_MEMBER:
-        return {...state, amount: String(payload.subscription_amount) }
+        return {...state, amount: String(payload.subscription_amount / 100) }
```

Comment:
After some reflection, I agree this is the right place for this division, though it doesn't look quite like it at first